tests/manual_e2e/basic_checks.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Helper function - prints an error message and exits
+exit_error() {
+    echo "Error: $*"
+    exit 1
+}
+
+echo "Checking if loki app is in deployed state"
+
+deployed=$(kubectl get app -n giantswarm loki -o yaml | yq .status.release.status)
+
+[[ "$deployed" != "deployed" ]] \
```

Comment:
```suggestion
status=$(kubectl get app -n giantswarm loki -o yaml | yq .status.release.status)

[[ "$status" != "deployed" ]] \
```

Reply:
Used `appStatus` to be even more precise :)